.travis.yml

```diff
@@ -0,0 +1,26 @@
+branches:
+  only:
+  - master
+sudo: required
+services: docker
+before_script:
+- echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+install:
+- sudo add-apt-repository -y ppa:duggan/bats > /dev/null 2>&1
+- sudo apt-get update
+- sudo apt-get install bats
+script:
+- make build
+- make test
+before_deploy:
+- make push
+deploy:
+  provider: releases
+  name: "$GIT_TAG"
```

Comment:
Is this a manual setting?

Reply:
Could be, I wasn't sure how we would version releases. I guess I could still generate a sqrl tag for the github release?